tests/llm_test.py

```diff
@@ -253,6 +253,22 @@ def test_validate_call_params_no_response_format():
     llm._validate_call_params()
 
 
+@pytest.mark.vcr(filter_headers=["authorization"])
```

Comment:
Can you please run this command on your shell to run test over this file? This will generate a couple of cassettes files
```
pytest -k llm_test
```